applications/virtual-fly-brain/client/src/components/Layout.js

```diff
@@ -97,16 +97,12 @@ const MainLayout = () => {
         }}
       >
         {tab.includes(0) && (
-          <Images />
+          <TermInfo />
```

Comment:
we should have instances of flexlayout/layout manager inside each tab, not the components hardcoded.